README.md

```diff
@@ -1,166 +1,132 @@
-1.  Create a pull request for review.
-
-### Post-merge steps
-
-After the first PR for your module is merged, follow these post-merge steps:
-
-1.  Create a PR to enable the upgrade test by removing the `t.Skip` line in `tests/pr_test.go`.
-
-<!-- Remove the content in this previous H2 heading -->
-## Reference architectures
-
-<!--
-Add links to any reference architectures for this module.
-(Usually in the `/reference-architectures` directory.)
--->
+|Profile|Category|ID|Description|
+|---|---|---|---|
+|NIST|AC-6|AC-6| Employ the principle of least privilege, allowing only authorized accesses for users. |
+|NIST|CP-9|CP-9(a) & CP-9(d)| Conduct backups of user-level information contained in system components and Protect the confidentiality, integrity, and availability of backup information. |
+|NIST|CP-10|CP-10| Provide for the recovery and reconstitution of the system to a known state after a disruption, compromise, or failure.|
+|NIST|CP-10(2)|CP-10(2)| Implement transaction recovery for systems that are transaction-based with the support of Point In Time Recovery (PITR)|
+|NIST|SC-5|SC-5| Limit the effects of denial-of-service events.|
+|NIST|SC-7|SC-7(3)| Limit the number of external network connections to the system.|
+|NIST|SC-7|SC-7(4)(c)| Implement a managed interface for each external telecommunication service.|
+|NIST|SC-7|SC-7(5)| Deny network communications traffic by default and allows network communications traffic by exception.|
+|NIST|SC-8|SC-8| Protect the confidentiality and integrity of data during transmission.|
+|NIST|SC-13|SC-13| Cryptography is implemented to support a variety of security solutions.|
+|NIST|SC-28|SC-28| Protect the confidentiality and integrity of data at rest.|
+|NIST|SC-28(1)|SC-28(1)| Implement cryptographic mechanisms to prevent unauthorized disclosure and modification of the information at rest.|
 
 ## Usage
 
-<!--
-Add an example of the use of the module in the following code block.
-
-Use real values instead of "var.<var_name>" or other placeholder values
-unless real values don't help users know what to change.
--->
-
-```hcl
-
+IBM Cloud Databases supports only Key Protect encryption for backups, not Hyper Protect Crypto Services. If you enable key management encryption and no value is passed for 'backup_encryption_key_crn', the value of 'kms_key_crn' is used. And if a HPCS value is set for `kms_key_crn`, the database backup encryption uses the default encryption keys. For more information, see [Hyper Protect Crypto Services Integration](https://cloud.ibm.com/docs/cloud-databases?topic=cloud-databases-hpcs) in the IBM Cloud Documents.
+
+```terraform
+provider "ibm" {
+  ibmcloud_api_key = "XXXXXXXXXX"
+  region           = "us-south"
+}
+
+module "redis" {
+  # replace "main" with a GIT release version to lock into a specific release
+  source            = "git::https://github.com/terraform-ibm-modules/terraform-ibm-icd-redis?ref=main"
```

Comment:
Can you update this to the hashicorp syntax (It will be published to hashicorp as soon as we have the first release)